app/views/home/index.html.erb

```diff
@@ -1,30 +1,28 @@
-<section class="hero is-white is-halfheight">
+<section class="hero is-small is-white is-halfheight">
   <div class="hero-body">
     <div class="container">
       <div class="columns is-vcentered reverse-columns">
```

Comment:
remove reverse-columns?

Reply:
remember in mobile text comes first

Reply:
ok, `reverse-columns` was redefined to `stacked-elements`